Cargo.toml

```diff
@@ -65,8 +65,6 @@ exclude = [
     "ffi/ecma402",
     # GNaw needs to be run from outside the workspace for features to work
     "ffi/gn",
-    # Special linker flags shouldn't overwrite the top-level target directory
-    "ffi/tinywasm",
```

Comment:
Issue: This was a problem when I was working on `tinywasm`. It has a custom linker script that Cargo doesn't know about, and it can mess things up in the top-level target directory. It's safer and easier if tinywasm is built in its own target directory.

Reply:
I removed the `tinywasm` crate and `build.sh` is just building `icu_capi_dylib` with a local target directory.

Reply:
https://github.com/unicode-org/icu4x/pull/3036/files#diff-1b2d115295f19607857a16c66f91306b0e4cc7f9e9481ce42db0ca7c10e670cf